package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-create-ui",
-  "version": "2.0.22",
+  "version": "2.0.23",
```

Comment:
Note to all (including me): let's bump this to 2.1 or 3 next time we release something big.

And/or, what would you guys think to change version to yyyy.mm.dd ?

cc @thorwolpert

Reply:
updated to `2.1.0`